src/app/modules/project/add/manual/manual.component.html

```diff
@@ -84,6 +83,30 @@ <h3>Project icon</h3>
       </div>
     </form>
 
+    <form [formGroup]="newCallToActionForm" class="project-form">
+
+      <div class="row">
+        <div class="col-12">
+          <h3>Call to action</h3>
+        </div>
+        <div class="col-6">
+          <div>
+            <h3>Text</h3>
```

Comment:
Just like with the other 'sections' like "Add project collaborators", there's a margin/padding between the the header and the actual content. 
I think it might be best to add it here as well.

Reply:
Please note that this feedback is applicable to editing a project as well.